Makefile

```diff
@@ -4,4 +4,5 @@ run:
 	@./00-setup-demo/setup.sh --offline-mode 1
 
 teardown:
-	kind delete cluster --name aws-demo
+	kind delete cluster --name ocm-demo
+	rm -r 00-setup-demo/charts/telepresence/
```

Comment:
this has been removed from the repo so is not necessary to add to makefile.
```suggestion
```

Reply:
it is actually pulled here https://github.com/open-component-model/demo-secure-delivery/blob/main/00-setup-demo/lib.sh#L325 and is not removed after cleanup

Reply:
then we should remove the pull I would suggest.

Reply:
I think nginx-ingress is intalled via the manifest here:
https://github.com/open-component-model/demo-secure-delivery/blob/main/00-setup-demo/lib.sh#L77-L83